okapi-core/src/test/java/org/folio/okapi/util/ModuleCacheTest.java

```diff
@@ -0,0 +1,350 @@
+package org.folio.okapi.util;
+
+import io.vertx.core.http.HttpMethod;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import org.folio.okapi.bean.InterfaceDescriptor;
+import org.folio.okapi.bean.ModuleDescriptor;
+import org.folio.okapi.bean.ModuleInstance;
+import org.folio.okapi.bean.RoutingEntry;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.aggregator.ArgumentsAccessor;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ModuleCacheTest {
+  @ParameterizedTest
+  @CsvSource({
+      "/, /",
+      "/{id}, /",
+      "{id},",
+      "/a/b{id}, /a/",
+      "/a/*c, /a/",
+      "/a/b*, /a/",
+      "/a/b/, /a/b/",
+      "/a/b, /a/b",
+      "/a/{id}, /a/",
+  })
+  void testGetPatternPrefix(ArgumentsAccessor accessor) {
+    RoutingEntry routingEntry = new RoutingEntry();
+    routingEntry.setPathPattern(accessor.getString(0));
+    String expect = accessor.getString(1);
+    if (expect == null) {
+      expect = "";
+    }
```

Comment:
You can directly provide an empty string when you use CsvSource' quoting:
`"{id}, ''"`

Reply:
ok